manifest.toml

```diff
@@ -57,13 +57,13 @@ ram.runtime = "50M"
     [resources.sources]
 
         [resources.sources.main]
-        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_26.x_Alnilam/26.2/tiki-26.2.tar.gz/download"
-        sha256 = "d85d513313e2377ecc2fe03f471b053727cb7f9a40143192f95191891557dabf"
+        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_27.x_Miaplacidus/27.0/tiki-27.0.tar.gz/download"
+        sha256 = "bcc79422239e9b17846082bec4341326518e95a5c058307e61933c2382245dbb"
         format = "tar.gz"
 
         [resources.sources.lts]
-        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_24.x_Wolf_359/24.6/tiki-24.6.tar.gz/download"
-        sha256 = "1dc1a3f0236c79287b14bfe45eb789e8b0e5c89d2a9089bab496c61a2c73a9f0"
+        url = "https://sourceforge.net/projects/tikiwiki/files/Tiki_27.x_Miaplacidus/27.0/tiki-27.0.tar.gz/download"
```

Comment:
Folks that choose LTS want stability. I would not replace 24.6 LTS with 27.0 LTS. I would wait for for 27.2 or 27.3.

Reply:
Sounds good.
Let's do this:
Now:
LTS cycle: 24.x (no change)
regular cycle: 26.x -> 27.x

And when 28.0 is released:
LTS cycle: 24.x -> 27.x
regular cycle: 27.x -> 28.x

I want to update the code and push the update to this PR.